qutip/solver/result.py

```diff
@@ -368,6 +374,29 @@ def __repr__(self):
     def expect(self) -> list[ArrayLike]:
         return [np.array(e_op) for e_op in self.e_data.values()]
 
+    def plot_expect(self):
+        """
+        Optional dependency matplotlib is REQUIRED.
+        Look at the expectation value(s) from evolution. Return a matplotlib
+        Figure instance and the associated array of Axes instance(s) for 
+        the expectation of each op.
```

Comment:
Could you reword this?

Also put the `matplotlib` requirement inside a `Notes` section. Look at the `add` method of `Result` for an example.